webpack.config.js

```diff
@@ -16,10 +17,23 @@ var config = require( './server/config' ),
  * Internal variables
  */
 var CALYPSO_ENV = process.env.CALYPSO_ENV || 'development',
+	AUTOOPEN = process.env.AUTOOPEN || 'false',
 	PORT = process.env.PORT || 3000,
 	jsLoader,
 	webpackConfig;
 
+function OpenCalypsoInBrowser() {
+	this.isFirstBuild = true;
+}
+OpenCalypsoInBrowser.prototype.apply = function( compiler ) {
+	compiler.plugin( 'done', function() {
+		if ( this.isFirstBuild ) {
+			open( 'http://calypso.localhost:3000' );
+			this.isFirstBuild = false;
+		}
+	}.bind( this ) );
+};
```

Comment:
Couldn't we just make this definition inside the parent function without modifying the prototype since it's all in one place? That would let us get rid of the bind too I think.

```js
function OpenCalypsoInBrowser() {
	let isFirstBuild = true;

	const apply = compiler => compiler.plugin( 'done', () => {
		if ( ! isFirstBuild ) { return; }

		isFirstBuild = false;
		open( … );
	} );
}
```